ui/admin/app/components/form/onboarding/index.js

```diff
@@ -0,0 +1,32 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class FormOnboardingComponent extends Component {
+  // =properties
+  @tracked targetAddress;
+  @tracked targetAddressInvalid = false; // change to method
+  @tracked targetPort;
+  @tracked targetPortInvalid = false; // change to method name
+
+  // =actions
+  @action
+  // change name to formSubmit
+  submit() {
```

Comment:
curious, what are these comments for? Are we changing these in the future?

Reply:
@DhariniJeeva sorry... rushing it = I always forget something 😅 Deleted all unnecessary comments